aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -61,34 +84,58 @@ module aptos_framework::staking_config {
         validate_required_stake(minimum_stake, maximum_stake);
 
         assert!(recurring_lockup_duration_secs > 0, error::invalid_argument(EZERO_LOCKUP_DURATION));
-        assert!(
-            rewards_rate_denominator > 0,
-            error::invalid_argument(EZERO_REWARDS_RATE_DENOMINATOR),
-        );
+
         assert!(
             voting_power_increase_limit > 0 && voting_power_increase_limit <= 50,
             error::invalid_argument(EINVALID_VOTING_POWER_INCREASE_LIMIT),
         );
 
-        // `rewards_rate` which is the numerator is limited to be `<= MAX_REWARDS_RATE` in order to avoid the arithmetic
-        // overflow in the rewards calculation. `rewards_rate_denominator` can be adjusted to get the desired rewards
-        // rate (i.e., rewards_rate / rewards_rate_denominator).
-        assert!(rewards_rate <= MAX_REWARDS_RATE, error::invalid_argument(EINVALID_REWARDS_RATE));
-
-        // We assert that (rewards_rate / rewards_rate_denominator <= 1).
-        assert!(rewards_rate <= rewards_rate_denominator, error::invalid_argument(EINVALID_REWARDS_RATE));
-
         move_to(aptos_framework, StakingConfig {
             minimum_stake,
             maximum_stake,
             recurring_lockup_duration_secs,
             allow_validator_set_change,
-            rewards_rate,
-            rewards_rate_denominator,
+            _rewards_rate: 0,
```

Comment:
Can you leave comments here explaining that this is set to 0 because it's been deprecated and replaced with the other struct?